packages/expo-font/src/__tests__/Font-test.native.ts

```diff
@@ -1,3 +1,5 @@
+import { Platform } from 'react-native';
```

Comment:
Shouldn't this be imported by `expo-modules-core`?

Reply:
I think it doesn't matter much as long as we don't need to use properties that are not available in Platform from `react-native` that are in `expo-modules-core`. I'll change it though, for better consistency. Thanks for pointing this out!